src/Components/Components/src/ComponentActivator.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components
+{
+    /// <summary>
+    /// Default implementation of component activator.
+    /// </summary>
+    public class ComponentActivator : IComponentActivator
```

Comment:
For consistency with other parts of the framework, can you rename this to `DefaultComponentActivator`?